src/java/fr/paris/lutece/plugins/wiki/web/WikiApp.java

```diff
@@ -456,103 +466,260 @@ public XPage getModifyTopic( HttpServletRequest request ) throws SiteMessageExce
         {
             topic = getTopic( request, strPageName, MODE_EDIT );
         }
-
+        // get last user present on modify page for this topic
+        String lastUser =    topic.getModifyPageOpenLastBy( );
+        Timestamp lastDate = topic.getModifyPageOpenAt( );
+        // if it's been less than 17 seconds since the last user, we cannot edit

```

Comment:
The topic is protected during only the first 17 s ? what's the point ?

Reply:
When a user is editing a page, every 15 second the timestamps and a string (firstname_lastname) are saved. So if a user wants to edit the page, and so far it's been less than 17 second than a previous user was on the page, that user is redirected to a page saying that this last user is editing. 
So it 's a bit arbitrary, I added 2 second in case of bad connection...